docs/docs/quickstart.md

```diff
@@ -182,17 +188,19 @@ Our two options for seeding a Postgres database are:
 1. Making a sequence of PSQL commands via the `psql` binary
 1. Using `pg_restore` to load a package of data
 
-Both are possible in Kurtosis, but for this tutorial we'll do the second one using a seed data TAR of DVD rental information, [courtesy of postgresqltutorial.com](https://www.postgresqltutorial.com/postgresql-getting-started/postgresql-sample-database/). 
+Both are possible in Kurtosis, but for this tutorial we'll use `pg_restore` to seed our database with a TAR of DVD rental information, [courtesy of postgresqltutorial.com](https://www.postgresqltutorial.com/postgresql-getting-started/postgresql-sample-database/). 
 
-Normally seeding a database would require downloading the seed data to your machine, starting Postgres, and writing a pile of Bash to copy the seed data to the Postgres server and run a `pg_restore`. If you forgot to check if the database is available, you may get flakes when you try to use the seeding logic in a test. 
+##### Without Kurtosis
```

Comment:
Does this level of header actually work?

Reply:
Would prefer bolded text then, because headers go into GH's autogenerated table of contents